### PR TITLE
[MODULAR] [READY][FIXES] Cargodise Lost fixes

### PR DESCRIPTION
**Pull request by ForrestWick**
## About The Pull Request

An upstream PR broke some of the item spawns, so this will fix those and make the power work properly, as well as renaming the spawners so that they don't share a space with the black market cryopod. 

Loot spawners are now placed again, power issues fixed, added a couple jetpacks for the salvage guys to use, and gave the guys a proper techfab as well so they can construct a teleporter and trade with the station.

## How This Contributes To The Skyrat Roleplay Experience

Fixing a ghost role I made

## Changelog

:cl:
fix: Made it so the cargodise lost ruin is on one single area with a single APC, meaning power issues are solved. The SMES and APC have been moved apart as well to help with this. Loot spawners have also been re-added.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/mapping/code/datums/shuttles.dm [hunk 1]
```diff
@@ -43,3 +43,30 @@
...
+	jumpto_ports = list("freighter" = 1, "whiteship_home" = 1)
+	view_range = 0
+	x_offset = 2
+	y_offset = 0
+
+
+/obj/item/circuitboard/computer/cargodise
+	name = "Freighter Salvage Vessel (Computer Board)"
+	greyscale_colors = CIRCUIT_COLOR_GENERIC
+	build_path = /obj/machinery/computer/shuttle/cargodise
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/mapping/code/datums/shuttles.dm]: New line

File: modular_skyrat/modules/mapping/code/game/objects/structures/ghost_role_spawners.dm [hunk 2]
```diff
@@ -281,22 +292,6 @@
...
-/area/ruin/space/has_grav/deepstorage/lostcargomine
-	name = "Freighter Ship Mining Room"
-	icon_state = "yellow"
-
-/area/ruin/space/has_grav/deepstoragelostcargoengine
-	name = "Freighter Ship Maintenance"
-	icon_state = "engine"
+/area/shuttle/cargodise
+	name = "Salvage Vessel"
+	requires_power = FALSE
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/mapping/code/game/objects/structures/ghost_role_spawners.dm]: new line

**Comment by Gandalf2k15** [comment]
Map merge it :)

**Comment by ForrestWick** [comment]
> Map merge it :)

Cant cause I'm at work

**Pull request merged**